lib/grammers-client/src/client/bots.rs

```diff
@@ -153,27 +153,29 @@ impl Client {
         InlineResultIter::new(self, bot.into(), query)
     }
 
+    /// Edits an inline message sent by a bot.
+    ///
+    /// Similar to [`Client::send_message`], advanced formatting can be achieved with the
+    /// options offered by [`InputMessage`].
+    ///
+    /// [`InputMessage`]: crate::InputMessage
     pub async fn edit_inline_message<M: Into<InputMessage>>(
         &self,
         message_id: tl::enums::InputBotInlineMessageId,
         input_message: M,
     ) -> Result<bool, InvocationError> {
         let message: InputMessage = input_message.into();
         let entities = parse_mention_entities(self, message.entities);
-        let dc_id = message_id.dc_id();
         let result = self
-            .invoke_in_dc(
```

Comment:
Would be good to make sure with messages with media in other DCs.

Reply:
Done